pyronear/datasets/openfire.py

```diff
@@ -42,13 +42,16 @@ class OpenFire(VisionDataset):
     seed = 42
 
     def __init__(self, root, train=True, transform=None, target_transform=None,
-                 download=False, threads=16, valid_pct=None):
+                 download=False, threads=16, valid_ratio=None):
         super(OpenFire, self).__init__(root, transform=transform,
                                     target_transform=target_transform)
         self.train = train  # training set or test set
 
         if download:
-            self.download(threads, valid_pct)
+            if isinstance(valid_ratio, (float, int)) and not (0 <= valid_ratio <= 1):
```

Comment:
Shouldn't it be this line be the following instead ?
```
if not isinstance(valid_ratio, (float, int)) or not (0 <= valid_ratio <= 1):
```
Otherwise a `str` object would be passed as `valid_ratio` to `download`. An alternative would be to put the type checking back in  `download`.